code/modules/mob/living/simple_animal/hostile/megafauna/megafauna.dm

```diff
@@ -34,6 +36,16 @@
 	else
 		..()
 
+/mob/living/simple_animal/hostile/megafauna/proc/grant_achievement()
```

Comment:
May want to make it so that the person has to have actually attacked the monster in say...a 30 second time frame for this to count.